SampleScripts/ppm/passwordpolicymanager.ps1

```diff
@@ -0,0 +1,1456 @@
+	if (!$checkReturn) {
+		$errorCount += 1
+	}
+	$checkReturn = checkRange -name "passwdMaxPreviousCharactersReused" -value $CommonValues."passwdMaxPreviousCharactersReused" -minRange 0 -maxRange 9999 -required $false
+	if (!$checkReturn) {
+		$errorCount += 1;
+	}
+	$checkReturn = checkRange -name "passwdAttemptsIntervalInMins" -value $CommonValues."passwdAttemptsIntervalInMins" -minRange 0 -maxRange 9999 -required $false
+	if (!$checkReturn) {
+		$errorCount += 1;
+	}
+	$checkReturn = checkRange -name "passwdUnlockIntervalInMins" -value $CommonValues."passwdUnlockIntervalInMins" -minRange 0 -maxRange 9999 -required $false
+	if (!$checkReturn) {
+		$errorCount += 1;
+	}
+	$checkReturn = checkRange -name "tempPasswdLifetimeInHour" -value $CommonValues."tempPasswdLifetimeInHour" -minRange 1 -maxRange 9999 -required $false
+	if (!$checkReturn) {
+		$errorCount += 1;
+	}
+	$checkReturn = checkRange -name "passwdExpReminderInDay" -value $CommonValues."passwdExpReminderInDay" -minRange 1 -maxRange 9999 -required $false
+	if (!$checkReturn) {
+		$errorCount += 1;
+	}
+	$checkReturn = checkRange -name "passwdExpReminderNotificationFrequencyInDay" -value $CommonValues."passwdExpReminderNotificationFrequencyInDay" -minRange 1 -maxRange 9999 -required $false 
+	if (!$checkReturn) {
+		$errorCount += 1;
+	}
+	if ($errorCount -gt 0) {
+		Write-LogMessage -Type ERROR -Message "There are errors in standard configuration Json please see above for more details"  -Colour Red
+		Write-LogMessage -Type ERROR -Message "Total error count = $errorCount" -Colour Red
+		Exit
+	}
+	
+	Write-LogMessage -type INFO -Message "Importing standard configuration setting from $($ppmVariables.commonPolicyFilePath)..."
+	parseProductTypeESXiConfiguration -productConfiguration $ppmStandardConfiguration -ppmStandardConfigValues $ppmStandardConfigValues
+	parseProductTypeVCConfiguration -productConfiguration $ppmStandardConfiguration -ppmStandardConfigValues $ppmStandardConfigValues
+	parseProductTypeSSOConfiguration -productConfiguration $ppmStandardConfiguration -ppmStandardConfigValues $ppmStandardConfigValues
+	parseProductTypeNSXMgrConfiguration -productConfiguration $ppmStandardConfiguration -ppmStandardConfigValues $ppmStandardConfigValues
+	parseProductTypeNSXEdgeConfiguration -productConfiguration $ppmStandardConfiguration -ppmStandardConfigValues $ppmStandardConfigValues
+	parseProductTypeWSAConfiguration -productConfiguration $ppmStandardConfiguration -ppmStandardConfigValues $ppmStandardConfigValues
+	Write-LogMessage -type INFO -Message "Importing standard configuration setting from $($ppmVariables.commonPolicyFilePath)...done"
+}
+
+#############################################################################
+####  Get-EnvironmentPasswordPolicyDetail will retrieve password policy #####
+####  of each components under a workload domain                        #####
+Function GetEsxiPasswordPolicy {
+	Param (
```

Comment:
Formatting, all Parameters in line (could be web review)

Reply:
addressed in 9ee41934dd0ee5a302dd128339b72e2f82898175